src/PrestaShopBundle/Form/Admin/Product/ProductQuantity.php

```diff
@@ -124,7 +124,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'options' => array(),
                 'locales' => $this->locales,
                 'hideTabs' => true,
-                'label' => $this->translator->trans('Label when out of stock', array(), 'Admin.Catalog.Feature'),
+                'label' => $this->translator->trans('Displayed text when out of stock', array(), 'Admin.Catalog.Feature'),
```

Comment:
Thanks! I updated it in another PR https://github.com/PrestaShop/PrestaShop/pull/5901